src/components/featured-products-section/featured-products-section.module.scss

```diff
@@ -0,0 +1,34 @@
+.root {
+    margin: 0 -2%;
+    padding: 75px 2%;
+}
+
+.header {
+    display: grid;
+    grid-auto-columns: minmax(0, 1fr);
+    grid-auto-flow: column;
+    align-items: center;
+    gap: 20px;
+    width: 100%;
```

Comment:
What are the auto columns for? You have a fixed number of elements in the header.